code/__DEFINES/logging.dm

```diff
@@ -1,54 +1,54 @@
 //Investigate logging defines
```

Comment:
now that i think about it, I dont think mentor logs are actually visible. maybe its worth adding